api/src/main/scala/hmda/api/http/HttpApi.scala

```diff
@@ -25,6 +41,37 @@ trait HttpApi extends HmdaApiProtocol {
           ToResponseMarshallable(status)
         }
       }
-    }
+    } ~
+      path("upload") {
+        post {
```

Comment:
No particular reason, it's just a higher level API. As it's implemented it would be equivalent, this way it's more explicit and gives more low level control